erasure-readfile_test.go

```diff
@@ -359,3 +362,65 @@ func TestErasureReadFileOffsetLength(t *testing.T) {
 		}
 	}
 }
+
+// Test erasureReadFile with random offset and lengths.
+// This test is t.Skip()ed as it a long time to run, hence should be run
+// explicitly after commenting out t.Skip()
+func TestErasureReadFileRandomOffsetLength(t *testing.T) {
+	// Comment the following line to run this test.
+	t.SkipNow()
+	// Initialize environment needed for the test.
+	dataBlocks := 7
```

Comment:
Do this for various dataBlocks one for each  i.e 2x2 (100), 3x3(100), 4x4(100), 5x5(100), 6x6(100), 7x7(100), 8x8(100) -- 100 is iterations here.